packages/ripple-address-codec/HISTORY.md

```diff
@@ -15,25 +15,6 @@
 * Eliminates 4 runtime dependencies: `base-x`, `base64-js`, `buffer`, and `ieee754`.
 * Execute test in a browser in addition to node
 
-## 5.0.0 Beta 1 (2023-11-30)
```

Comment:
This still feels relevant.  There were releases.  It also shows we released betas.

Reply:
It's not useful in the changelog IMO - it's already in the release list.

Reply:
I personally find negative value in having beta info in changelogs, even when looking at other packages. I think it just clutters up the file. But if you disagree I'm happy to roll back this part of the change.